cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -66,11 +68,16 @@ type ClusterInterrogator interface {
 type Client struct {
 	Endpoints []string
 	TLS       *tls.Config
+
+	newEtcdClientFn func(cfg clientv3.Config) (*clientv3.Client, error)
```

Comment:
i think we can drop this, given the comments about tests.

Reply:
Fair enough - dropped!